plugins/modules/sonic_route_maps.py

```diff
@@ -0,0 +1,1608 @@
+                  - Tag (IMET) route, or prefix route
+                type: str
+                choices:
+                  - macip
+                  - multicast
+                  - prefix
+              vni:
+                description:
+                  - VNI ID to be checked for a match; specified by a value in the
+                  - range 1-16777215
+                type: int
+          ext_comm:
+            description:
+              - Name of a configured BGP 'extended community' to be checked for
+              - a match with the target route
+            type: str
+          interface:
+            description:
+              - Next hop interface name (type and number) to be checked for a
+              - match with the target route. The interface type can be any
+              - of the following; 'Ethernet/Eth' interface or sub-interface,
+              - "'Loopback' interface, 'PortChannel' interface or"
+              - "sub-interface, 'Vlan' interface."
+            type: str
+          ip:
+            description:
+              - IP addresses or IP next hops to be checked for a match with the
+              - target route
+            type: dict
+            suboptions:
+              address:
+                description:
+                  - name of an IPv4 prefix list containing a list of address
+                  - prefixes to be checked for a match with the target route
+                type: str
+              next_hop:
+                description:
+                  - name of a prefix list containing a list of next-hop
+                  - prefixes to be checked for a match with the target route
+                type: str
+          ipv6:
+            description:
+              - IPv6 addresses to be checked for a match with the
+              - target route
+            type: dict
+            suboptions:
+              address:
+                description:
+                  - name of an IPv6 prefix list containing a list of address
+                  - prefixes to be checked for a match with the target route
+                type: str
+                required: true
+          local_preference:
+            description:
+              - local-preference value to be checked for a match with the
+              - target route. This is a value in the range 0-4294967295.
+            type: int
+          metric:
+            description:
+              - metric value to be checked for a match with the target route.
+              - This is a value in the range 0-4294967295.
+            type: int
+          origin:
+            description:
+              - BGP origin to be checked for a match with the target route
+            type: str
+            choices:
+              - egp
+              - igp
+              - incomplete
+          peer:
+            description:
+              - BGP routing peer/neighbor required for a matching route
```

Comment:
Please specify that the peer options are mutually exclusive.

Reply:
Done.